CRM/Contribute/Form/ContributionPage/Amount.php

```diff
@@ -372,6 +372,11 @@ public static function formRule($fields, $files, $self) {
         $errors['pay_later_receipt'] = ts('Please enter the instructions to be sent to the contributor when they choose to \'pay later\'.');
       }
     }
+    else {
+       if (empty($fields['payment_processor'])) {
+         $errors['payment_processor'] = ts('You have listed fixed contribution options or selected a price set, but no payment option has been selected. Please select at least one payment processor and/or enable the pay later option.');
+        }
+      }
```

Comment:
The description of this PR refers to "Event registration Page" configuration but you are also making changes to the "Contribution Page" configuration.  This change makes sense, but can you please update the PR description?

Reply:
Also, if I uncheck `"Contribution Amounts section enabled"` and have no payment processors selected I still cannot save the contribution page.